src/test/java/hudson/scm/SVNWorkingCopyTest.java

```diff
@@ -40,10 +40,12 @@
 import org.tmatesoft.svn.core.wc2.SvnGetStatus;
 
 import java.io.File;
+import org.junit.Ignore;
 
 /**
  * @author schristou88
  */
+@Ignore("TODO org.tmatesoft.svn.core.SVNException: svn: E175002: PROPFIND of '/trunk/hudson/test-projects/trivial-ant': 405 Method Not Allowed (https://svn.jenkins-ci.org)")
```

Comment:
I think you're getting this error because https://svn.jenkins-ci.org doesn't exist anymore. Probably need to point it to a new subversion repository.